src/freenet/clients/http/QueueToadlet.java

```diff
@@ -44,10 +45,11 @@
 import freenet.client.async.PersistentJob;
 import freenet.client.async.TooManyFilesInsertException;
 import freenet.client.filter.ContentFilter;
-import freenet.client.filter.KnownUnsafeContentTypeException;
 import freenet.client.filter.FilterMIMEType;
+import freenet.client.filter.KnownUnsafeContentTypeException;
```

Comment:
The Date import is new, which is why I organized the imports to get it. This can be done in one step with Eclipse: After you typed code which uses a class which is not imported yet, you press CTRL+O. It then imports the new class, and also organizes the remaining imports.
I usually don't even notice doing this anymore.

I'd say the time which is saved by this is worth the occasional noise. We also get a gradual, implicit cleanup of imports by doing this frequently, similar to the gradual fixing of programming standards by applying them to new code. 
-> ACK to keep as is?